packages/core/src/components/TextField/Styled/Label.styled.tsx

```diff
@@ -30,7 +30,8 @@ export const Label = styled('label')<{ required: boolean; variant: 'outlined' |
     transform: translateY(-50%);
     opacity: 1;
     z-index: 1;
-    width: 100%;
+
+    /* width: 100%; */
```

Comment:
@jaredgiangrasso For the fusion label I see some trailing whitespace. Removing the width here fixes it, but I didn't want to remove without flagging for you first. Is the width here required for something else?